samples/simpleSubOrchestrator/profile.ps1

```diff
@@ -0,0 +1,23 @@
+# Azure Functions profile.ps1
+#
+# This profile.ps1 will get executed every "cold start" of your Function App.
+# "cold start" occurs when:
+#
+# * A Function App starts up for the very first time
+# * A Function App starts up after being de-allocated due to inactivity
+#
+# You can define helper functions, run commands, or specify environment variables
+# NOTE: any variables defined that are not environment variables will get reset after the first execution
+
+# Authenticate with Azure PowerShell using MSI.
+# Remove this if you are not planning on using MSI or Azure PowerShell.
+if ($env:MSI_SECRET) {
+    Disable-AzContextAutosave -Scope Process | Out-Null
+    Connect-AzAccount -Identity
+}
+
+# Uncomment the next line to enable legacy AzureRm alias in Azure PowerShell.
+# Enable-AzureRmAlias
+
+# You can also define functions or aliases that can be referenced in any of your PowerShell functions.
+Import-Module AzureFunctions.PowerShell.Durable.SDK -ErrorAction Stop
```

Comment:
I would leave an empty space for line 23. On line 24 add a relevant comment and add this `Import-Module AzureFunctions.PowerShell.Durable.SDK -ErrorAction Stop` to line 25. Otherwise, it is a bit confusing the comment above the import-module statement.

Reply:
Ah you're right, did not think of it. Done :)